vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/Analysis/ModuleAnalysisTests.cs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+namespace Microsoft.ReactNative.Managed.CodeGen.UnitTests.Analysis
+{
+  [TestClass]
+  public class ModuleAnalysisTests : AnalysysTestBase
+  {
+    [TestMethod]
+    public void NoAttirbuteIsNoModule()
+    {
+      var (codeAnalyzer, type) = AnalyzeFile(@"
+public class TestClass
```

Comment:
>pub [](start = 0, length = 3)

Consider adding indentations inside of the test strings.  I guess it must be parse-able anyway.